apps/dev/nextjs/auth.config.ts

```diff
@@ -39,7 +45,7 @@ export default {
     }),
     GitHub,
     Google,
-    Keycloak,
+    // Keycloak,
```

Comment:
I am getting the following error when running NextJS app via `pnpm run dev`:

```
 InvalidEndpoints: Provider "keycloak" is missing both `issuer` and `authorization` endpoint config. At least one of them is required. Read more at https://errors.authjs.dev#invalidendpoints
```

If it is related to my environment only let me know it and I will uncomment the line.